spec/bojangles_spec.rb

```diff
@@ -266,59 +270,24 @@
       end
     end
   end
-  describe 'message_html' do
-    context 'with error messages' do
-      it 'creates a message' do
-        Bojangles.stub(:message_list) do
-          'error message list'
-        end
-        message = <<~message.tr("\n", ' ')
-          This message brought to you by Bojangles, UMass Transit's
-          monitoring service for the PVTA realtime bus departures feed.
-        message
-        expect(Bojangles.message_html(%w[error1 error2], current: true))
-          .to include message + '<br>' + Bojangles.message_list
-      end
-    end
-  end
-  describe 'message_list' do
-    context 'with new error messages' do
-      it 'creates a message_list of given heading and error messages' do
-        heading = 'Bojangles has noticed the following errors:'
-        result = Bojangles.message_list(%w[error1 error2], current: true)
-        expect(result.first).to include heading
-        expect(result.last).to include 'error1'
-        expect(result.last).to include 'error2'
-      end
-    end
-    context 'with resolved error messages' do
-      it 'creates a message_list of given heading and error messages' do
-        heading = 'This error has been resolved:'
-        result = Bojangles.message_list(%w[error1 error2], current: false)
-        expect(result.first).to include heading
-        expect(result.last).to include 'error1'
-        expect(result.last).to include 'error2'
-      end
-    end
-  end
 
   describe 'compare_errors' do
     let(:message) { %w[apple banana cashew].join "\n" }
     let(:different_message) { %w[apple barley cashew].join "\n" }
     let(:equivalent_message) { %w[apple banana carrot].join "\n" }
 
     let(:result) { Bojangles.compare_errors current, old }
-    let(:current) { [message] }
+    let(:current) { [{ message: message }] }
     context 'messages differ by one of the first two lines' do
-      let(:old) { [different_message] }
+      let(:old) { { different_message => 123 } }
```

Comment:
Why did you use a hash rocket here? I'm honestly confused.

Reply:
The key is a variable.

Reply:
Ohhh